src/components/Editor/PreviewOptions.vue

```diff
@@ -42,20 +42,31 @@
 			@change="e => $emit('update:value', e.currentTarget.value)">
 			{{ t('text', 'Show link preview') }}
 		</NcActionRadio>
+		<NcActionSeparator />
+		<NcActionButton @click="e => $emit('update:value', 'delete-preview')">
+			<template #icon>
+				<DeleteIcon :size="20" />
+			</template>
+			{{ t('text','delete') }}
```

Comment:
https://docs.nextcloud.com/server/latest/developer_manual/design/foundations.html#wording

> For any “Delete” action, give context to what it will delete, like “Delete conversation” or “Delete user” so that it is clear specifically for this destructive action.